_posts/2019-02-05-configurable-builds-part-1.md

```diff
@@ -32,15 +32,16 @@ roadmap](https://bazel.build/roadmaps/configuration.html).
 
 ## History
 
-Bazel's a
+Bazel is a
 [powerful](https://docs.bazel.build/versions/master/bazel-vision.html) build
-tool that's especially suited for large,
-[multi-language](/2018/12/05/multilanguage-build-system.html) builds. But it
-was born out of Google, which historically wrote code for homogeneous fleets of
-Linux servers with little need for customization. What customization was needed
-was achieved with ad hoc flags and logic built
-[straight](https://source.bazel.build/bazel/+/144912e7b7a86b45e07f79e76f6fed20890acb36:src/main/java/com/google/devtools/build/lib/rules/cpp/CppOptions.java;l=258)
-into the build tool.
+tool that's especially suited for large codebases with
+[multiple languages](https://blog.bazel.build/2018/12/05/multilanguage-build-system.html)
+builds. But it grew out of Google, which historically wrote code for fleets of
```

Comment:
I think this extra "builds" should be removed.

Reply:
Also the "But" in the next sentence.

Reply:
Done on the extra "builds". 

I'm trying to set the next sentence up as a problem w.r.t the advertised "power".  I don't want it to read as if it's reinforcing the previous sentence - it's highlighting limitations that get in the way of that.

Reply:
SGTM.